### PR TITLE
Improved engine start, flyby sound, and other small things

### LINKED ISSUE #1319: [BUG] [SOUND] Fast circling of Seat belt sign should trigger sound twice
**Mod Version**

0.3.1 current master as of 13th Okt 20

**Describe the bug**

When circling the fasten seat belt sign fast it only plays its sound once.

**To Reproduce**

1. Circle the fasten seatbelt sign fast
2. 
3.

**Expected behavior**

It should trigger the fasten seatbelt sign sound twice.

**Actual behavior**

If you do it fast it triggers just once. If you do it slowly it works.

**References**

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by hotshotp**
Fixes #1319

## Summary of Changes

Improved engine startup on interior - made it even closer to the real thing
Improved engine startup on exterior - added treble/shshshshhshs sound that the engine makes on the exterior
Added new flyby sound layer - the swish-swoosh sound when the plane flies by
Added new random sample to the external power relay
Added new random sample to the avionics fan test so it now has two random samples
Added one second delay to the seatbelt sign, and now you can do quick double dings of the seatbelt sign
Improved rumble to the take-off thrust to be more "roary"
Made turbulence sound more sensitive

For later use by @Benjozork - new wwise events
firepb - red fire push button
firepbguard - red fire push button guard
mcdubuttons - mcdu button
revlever - reverse lever

since you work with templates and you told me not to touch them ill leave these for you later

## Additional context

Since i made turbulence more sensitive make sure it doesn't trigger on smooth turns and only on abrupt movements

Discord username (if different from GitHub):

Boris#9134

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 1]
```diff
@@ -188,241 +189,293 @@
...
+            <WwiseRTPC SimVar="TURB ENG N2" Units="percent" Index="1" RTPCName="SIMVAR_TURB_ENG_N2" />
+            <WwiseRTPC SimVar="TURB ENG N1" Units="percent" Derived="true" Index="1" RTPCName="SIMVAR_TURB_ENG_N1_DERIVED" />
+        </Sound>
+
+        <Sound WwiseEvent="StartVR" NodeName="REACTOR_RIGHT" FadeOutType="1" FadeOutTime="1" WwiseData="true" SimVar="TURB ENG N2" Units="percent" ViewPoint="Inside" Index="2" Continuous="true">
+            <Range LowerBound="1" />
+            <WwiseRTPC SimVar="TURB ENG N2" Units="percent" Index="2" RTPCName="SIMVAR_TURB_ENG_N2" />
+            <WwiseRTPC SimVar="TURB ENG N1" Units="percent" Derived="true" Index="2" RTPCName="SIMVAR_TURB_ENG_N1_DERIVED" />
+        </Sound>
+
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:406]:
```suggestion
```

**Review by Benjozork — APPROVED** [review]
LGTM

**Comment by ErickSharp** [comment]
Category: QA Trial
Name: 🎄🎅Erick🎄🎅 [Z-5]#6484
Date of testing: Current date 24/12/2020
Version of the sim: Version the tester uses
PR Tested: #2676 
Object of test: Improved engine-startup, fly-by, and other sounds

Testing technique: Started cold and dark and tested sounds in order of typical startup procedure.
Steps you've made to test the Pull Request:
1. Prepared cockpit and selected APU for startup.
2. Before APU finished startup, I turned external power on and off to confirm different sounds were played for power relay and fan test.
3. With power available, I flicked the seatbelt sign on and off twice to confirm several chimes were possible.
4. Started up engine and listened to it inside.
5. Shut down both engines, went outside, started engines and listened to it.
6. Took off, placed camera down the runway to listen to engine as it passed by.
7. Restarted takeoff roll and listened to the rotation sound.
8. While in the air at a stable altitude, I jerked the rudder and flight controls to hear the jostling noise.

Flight notes (If required): Testing done at Madeira, startup from cold and dark.
Issues (If Any): None
Overall Rating: Good
Conclusions of the testing: Changes to the sounds bring it closer to real footage I am able to find online. Things such as the bellow of the engine during startup, the improved deep rumble on rotation, and the jostling noise observed during turbulence fit nicely and make it feel natural. All of the sounds play correctly and at the right time. Overall good sound changes, and no major changes necessary.

**Comment by NickFarin** [comment]
Category: QA Trial
Name: Discord User: Nick F[Z+8] #5327
Date of testing: 24/12/2020
Version of the sim: 1.12.13.0
PR Tested: #2676 
Object of test: Improved engine-startup, fly-by, and other sounds
Testing technique: Spawned in at KLAS and tested sounds
Steps you've made to test the Pull Request:
1. Spawned at KLAS in the cold and dark config
2. Performed startup procedure and listened to the sounds connected to the startup procedure
3. Started APU and started engines and listened to the sounds related to the engine startup
4. After engine sounds tested, tested seatbelt signs and listened to the sounds f the seatbelt chimes
5. After seatbelt sounds tested, taxied to the runway and applied TOGA power ang listened to the engines from the cockpit and cabin

Flight notes (If required):

Issues (If Any): 
Overall Rating: Good
Conclusions of the testing: Startup sounds are better sounding due to the deeper rumble and other sounds are improved from the sounds from the older A32nx versions.

**Comment by DroteQ** [comment]
Category: QA Tester(maybe)
Name: DroteQ#2742
Date of testing: 24.12.2020 (Waiting for my Christmas bonus)
Version of the sim: 1.12.13.0 
PR Tested: #2676
Object of test: Testing the Start up of the engine on the ground and simulated engine failure, also testing the new sound layers added.

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. Turning on the APU to start the engine 2
2. After initial start up of the engine 2 turned of the APU
3. Starting the engine 1 with the engine 2 bleed air, engine 1 start up sequence did not initiate.
4. Turning on the APU again and starting the engine 1, work this time
5. Did an simulated engine failure, after that managed to start the engine 2 backup again and continue the flight.

Overall Rating: Good
The sound tweaks in make it sound more like the real Airbus, also the new turbulence sound is sounding perfect when flying in 
turbulence conditions. Overall nice add on the A32NX dint found any bugs. (except engine bleed air start, if its an one note sure)

**Comment by Lazarus320** [comment]
| Category: QA Tester  |
|----------------------|
| Name: Lazarus#4286   |
| Date of testing: 24/12 /2020 |
| Version of the sim: 1.12.13  |
| PR Tested: #2676          |
| Object of test: Engine start, flyby sound, and other small things. |

| Testing technique: Loaded c&d and every new sound or change individually.|
|-|

| Steps to test the Pull Request: |                                                  |
|-|-|
| 1 | New external power relay sound checked. https://user-images.githubusercontent.com/74972083/103070881-6cd6f000-45fd-11eb-8c5f-3db6afb2d31d.mp4 | 
|             2                  |  New avionics fan sample checked. https://user-images.githubusercontent.com/74972083/103071525-b7a53780-45fe-11eb-8da3-f1fe5dfdf391.mp4 |
|              3                   | One second delay to the seatbelt sign and double chime checked. https://user-images.githubusercontent.com/74972083/103074656-0b1a8400-4605-11eb-912c-d9cf8a491032.mp4 |
| 4 | Improved engine startup sound on exterior (treble) checked. https://user-images.githubusercontent.com/74972083/103077820-7ff0bc80-460b-11eb-912f-be04bca96140.mp4 |
| 5 | Improved interior engine startup sound exterior checked. https://user-images.githubusercontent.com/74972083/103078360-a5ca9100-460c-11eb-870c-e8f0f01cbc1f.mp4 |
| 6 | Flyby sound layer checked. https://user-images.githubusercontent.com/74972083/103080180-45d5e980-4610-11eb-9011-2690574da3b6.mp4 |
| 7 | Higher turbulence sound sensitivity checked. https://user-images.githubusercontent.com/74972083/103081590-360bd480-4613-11eb-81ba-4ba1600c2e03.mp4 |

| Flight notes: Tested using 7.1 surround |
| --- |

| Issues: N/A |
| --- |

| Overall rating: | Good |
|-|-|

| Conclusions of the testing: | Great addition especially with the variety of random sound samples, good job! |
|-|-|

**Pull request merged**